app/templates/components/rental-listing.hbs

```diff
@@ -1,6 +1,6 @@
 <article class="listing">
-  <a {{action 'toggleImageSize'}} class="image {{if isWide "wide"}}">
-    <img src="{{rental.image}}" alt="">
+  <a {{action 'toggleImageSize'}} class="image {{if isWide "wide"}}" aria-hidden="true" tabindex="-1">
```

Comment:
- Interactive elements are not supposed to be hidden- can you tell me what the purpose would be here? 
- Also, if you were to hide an element, you wouldn't need an alt attribute value on a child element.

Reply:
It was overlooked during copy & paste. I'll fix this and then get started on any edits to the tutorial